cypress/support/import-ui-test-data-to-firestore.js

```diff
@@ -0,0 +1,144 @@
+// eslint-disable-next-line
+const admin = require("firebase-admin");
+// eslint-disable-next-line
+const serviceAccount = require("./serviceAccountKey.json");
+
+/* DECIDE INTO WHICH FIRESTORE SHOULD DATA BE IMPORTED */
+let projectId = process.argv[2]; // [0] is node command, [1] is name of script, [2] is parameter
+if (projectId === undefined) {
+  // eslint-disable-next-line no-console
+  console.log("Importing data into 'Boxwise Feature Tests' firestore");
+  projectId = "boxwise-feature-tests";
+} else {
+  // eslint-disable-next-line no-console
+  console.log(`Importing data into firestore of project with ID: ${projectId}`);
+}
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: `https://${projectId}.firebaseio.com`
+});
+
+/* IMPORT DATA */
+const testOrgName = "browserTestBoxwiseUser";
+const testUserName = "browserTestBoxwiseUser";
+const testUserEmail = "browserTestBoxwiseUser@example.com";
+const testUserPwd = "browserTestBoxwiseUser";
+
+const categories = [
+  "Man",
+  "Woman",
+  "Adult",
+  "Boy",
+  "Girl",
+  "Child",
+  "Baby",
+  "Food",
+  "Hygiene",
+  "Other"
+];
+
+admin
+  .auth()
+  .listUsers()
+  .then(listUsersResult => {
```

Comment:
@DurkoMatko more a heads up than anything - you can avoid all the nesting (known as 'callback hell'!) by chaining the return values on then() instead - something like this: https://gist.github.com/jamescrowley/4cf7f86b48275ad51ee232fdb9e34b3e .

Reply:
You can also disable the ESLint no-console rule for the whole file, with
`
/* eslint-disable no-console */
`
at the top of the file

Reply:
Thx! I'll update the code to avoid the callback hell :) 

Btw, I'm finding pretty difficult to read the code when logs are split into several lines by linter. Shouldn't logs take only as small number of lines as necessary? If it's something we want to get rid of, I can take a look into ESlinter settings how to maybe define an exception for console.log statements.

Reply:
Fixed